insightfinder/assets/dashboards/ifdashboard.json

```diff
@@ -0,0 +1,276 @@
+{
```

Comment:
Please add an InsightFinder logo to your dashboard.

Can you also please change all spellings of "DataDog" to "Datadog" in both your tile and documentation: https://insightfinder.com/datadog-integration/?